app/lib/screens/HomeScreens/faq/Overview.dart

```diff
@@ -4,11 +4,21 @@ import 'package:effektio/widgets/FaqListItem.dart';
 import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
 import 'package:flutter/material.dart';
 
-class FaqOverviewScreen extends StatelessWidget {
+class FaqOverviewScreen extends StatefulWidget {
   final Client client;
-
   const FaqOverviewScreen({Key? key, required this.client}) : super(key: key);
 
+  @override
+  State<FaqOverviewScreen> createState() =>
+      _FaqOverviewScreenState(this.client);
+}
+
+class _FaqOverviewScreenState extends State<FaqOverviewScreen> {
+  final TextEditingController SearchController = TextEditingController();
```

Comment:
Maybe you forgot to change variable conventional name, it should start with smaller alphabet.